components/gender-toggler/GenderToggler.tsx

```diff
@@ -0,0 +1,46 @@
+import { TFunction } from 'next-i18next';
+import { Gender, Talent } from '../../lib/types';
+import OptionToggler, {
+  OptionTogglerProps,
+} from '../option-toggler/OptionToggler';
+
+interface GenderTogglerProps extends Partial<OptionTogglerProps> {
+  t: TFunction;
+  updateFunction: React.Dispatch<React.SetStateAction<Partial<Talent>>>;
+  input?: Gender;
+}
+
+export const GenderToggler = ({
```

Comment:
so in the end we will need either the GenderRadio or GenderSelector or GenderToggler component, dependent on the design, right?

Reply:
exactly

Reply:
thanks!